### ISSUE TITLE
HorizontaBarChartWithYAxis not working

### ISSUE THREAD

**Issue by VinhNguyenDucThuy** [JesperLekland/react-native-svg-charts#392]
### What is the problem?
HorizontaBarChartWithYAxis example have bug with

TypeError: TypeError: y.ticks is not a function. (In 'y.ticks(numberOfTicks)', 'y.ticks' is undefined)
- node_modules/react-native-svg-charts/src/y-axis.js:73:69 in render
- node_modules/react-native/Libraries/Renderer/oss/ReactNativeRenderer-dev.js:11581:21 in finishClassComponent
- ... 16 more stack frames from framework internals

and 

data[ index ].label undefined

### What platform?

* [x] iOS 
* [x] Android 

React Native version:
react-native-cli: 2.0.1
react-native: 0.59.8

react-native-svg-charts-version: 5.3.0

### Code to reproduce

https://github.com/JesperLekland/react-native-svg-charts-examples/blob/master/storybook/stories/bar-chart/horizontal-with-axis.js

```javascript
// put code here
```

**Comment by JonasDreessen** [comment]
y.ticks is undefined because scale={scale.scaleBand} needs to be ScaleBand. 
I have the same problem with the index.label is undefined

**Comment by usrbowe** [comment]
@VinhNguyenDucThuy I don't think this is an issue of library. The example is using `d3-scaleBand`, which doesn't generate `ticks` (source: https://github.com/d3/d3-scale/blob/master/src/band.js).

As the `d3Scale` repo says:
> Scales have no intrinsic visual representation. However, most scales can generate and format ticks for reference marks to aid in the construction of axes.



The default scale is `d3-scaleLinear`, which support `ticks` (source: https://github.com/d3/d3-scale/blob/master/src/linear.js#L9)


So basically my conclusion is that we should fix the example, and it's developers responsibility to use correct d3-scale function. Other way is to add check and throw error, if unsupported scale function is provided.
